README.md

```diff
@@ -9,10 +9,6 @@ This action processes maven surefire or failsafe XML reports on pull requests an
 
 ## Inputs
 
-### `github_token`
-
-**Required**. Usually in form of `github_token: ${{ secrets.GITHUB_TOKEN }}`.
```

Comment:
Keep this as optional field. People might still use their own custom tokens for some reason.

> Optional. Usually in form of `github_token: ${{ secrets.GITHUB_TOKEN }}`. The default is `${{ github.token }}`.

Reply:
updated! https://github.com/ScaCap/action-surefire-report/pull/52/commits/c716a352959254e682db0ca1548c24d466e48659